Grinnell-DB-iOS/ResultsViewController.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
+
+
+extension ResultsViewController: UIViewControllerPreviewingDelegate {
```

Comment:
If this is an extension/category, i think you should use the category file-name convention. Not sure if its different in swift. But I can also understand this as a stepping stone towards rewriting the entire controller in swift.